themes/default/content/blog/deploy-to-multiple-regions/index.md

```diff
@@ -0,0 +1,759 @@
+
+{{% chooser language "typescript,python,go,csharp,yaml" / %}}
+
+{{% choosable language typescript %}}
+
+```typescript
+const awsWest2 = new aws.Provider("aws-west-2", { region: "us-west-2" });
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+aws_west_2 = aws.Provider('aws-west-2', region='us-west-2')
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+awsWest2, err := aws.NewProvider(ctx, "aws-west-2", &aws.ProviderArgs{
+    Region: pulumi.String("aws-west-2")
+})
+if err != nil {
+    return err
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+var awsWest2 = new Aws.Provider("aws-west-2", new Aws.ProviderArgs
+{
+    Region = "us-west-2"
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+resources:
+    awsWest2:
+        type: pulumi:providers:aw
+        properties:
+            region: us-west-2
+```
+
+{{% /choosable %}}
+
+This configuration is _not_ used automatically, however. You must pass it to any resources you'd like to use it:
+
+{{% chooser language "typescript,python,go,csharp,yaml" / %}}
+
+{{% choosable language typescript %}}
+
+```typescript
+const buck = new aws.s3.Bucket("my-bucket", {
+    // ...
+}, { provider: awsWest2 });
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+buck = aws.s3.Bucket('my-bucket',
+    # ...
+    __opts__=pulumi.ResourceOptions(provider=aws_west_2)
+)
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+buck, err := s3.NewBucket("my-bucket",
+    &s3.BucketArgs{
+        // ...
+    },
+    pulumi.Provider(awsWest2)
+)
+if err != nil {
+    return err
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+var buck = new Aws.S3.Bucket("my-bucket",
+    new Aws.S3.BucketArgs
+    {
+        // ...
+    },
+    new ResourceArgs
+    {
+        Provider = awsWest2
+    }
+);
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+resources:
+    # ...
+    buck:
+        type: aws:s3:Bucket
+        properties:
+            # ...
+        options:
+            provider: ${awsWest2}
+```
+
+{{% /choosable %}}
+
+This overrides the default of using whatever was set at the CLI, and will instead use the programmatically configured settings. Note also that anything you can set at the CLI is available, not just the region. You can see the full settings available in the package's registry documentation for its `Provider` class (for instance, [AWS's]({{< relref "/registry/packages/aws/api-docs/provider" >}}).
+
+## Multi-Region Deployment In Action!
+
+The architecture for our multi-region setup is going to be very simple so we can focus on the essentials. We will provision a single RDS primary database in one region (us-east-1) and backup to a read replica in an entirely different region -- in fact, on an entirely different continent (eu-west-2)!
```

Comment:
so in this case, what if someone new to this doesn't find this simple, it could make them feel like this isn't for them. is there a more specific descriptor we can use here, maybe `short`, or possibly drop the entire first sentence, is it necessary if the following sentence is describing what is involved?

Reply:
Let me noodle on this.  I agree with your point.  The thing I was trying to get across is that, I'd expect _most_ people interested in this post to have the opposite feeling!  That is, maybe they wanted to loop over all available AWS regions, or something even more complex, and by seeing the overly simplistic "two databases in two regions," I worry they may conclude Pulumi can't handle their complex use case.  That said, I think I can phrase it differently.